src/hearings/viewModel/case-hearing-view.model.ts

```diff
@@ -0,0 +1,4 @@
+import { CaseHearingModel } from 'api/hearings/models/caseHearing.model';
+export interface CaseHearingViewModel extends CaseHearingModel {
```

Comment:
Could this file put under existing models folder? Reason:
1. This is still a model
2. Doesn't make sensen to create a new folder just for 1 new file inside